packages/charts/src/chart_types/goal_chart/renderer/canvas/connected_component.tsx

```diff
@@ -133,6 +135,7 @@ class Component extends React.Component<Props> {
           role="presentation"
         >
           <ScreenReaderSummary />
+          {semanticValues && semanticValues.length > 1 && <GoalSemanticDescription semanticValues={semanticValues} />}
```

Comment:
Will this be the same thing?

```suggestion
          {semanticValues?.length > 1 && <GoalSemanticDescription semanticValues={semanticValues} />}
```

Reply:
Actually, should we hide all of this logic in `GoalSemanticDescription`? (have that component do the conditional check so that we have a cleaner structure here)

Reply:
thank you! 250ab2c